packages/react-server-cli/src/findOptionsInFiles.js

```diff
@@ -13,12 +13,15 @@ const PACKAGE_JSON = "package.json";
 // the package.json. if it finds neither, it goes up a directory and looks again.
 // it returns the contents of the first config file found; it never merges multiple
 // configurations.
-export default (dir = process.cwd()) => {
+export default (args = process.argv) => {
+	let dir = process.cwd();
```

Comment:
I removed it because it wasn't used in any of the function calls. I can add it back though!